examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -100,12 +116,40 @@ class GalleryDrawer extends StatelessWidget {
             applicationLegalese: '© 2016 The Chromium Authors',
             aboutBoxChildren: <Widget>[
               new Padding(
-                padding: const EdgeInsets.only(top: 24.0, bottom: 8.0),
-                child: new Center(child: new CircularProgressIndicator())
-              ),
-              new Text('Awaiting fix for issue 4512.', textAlign: TextAlign.center),
+                padding: const EdgeInsets.only(top: 24.0),
+                child: new RichText(
+                  text: new TextSpan(
+                    children: <TextSpan>[
+                      new TextSpan(
+                        style: aboutTextStyle,
+                        text: "Flutter is an early-stage, open-source project to help "
+                        "developers build high-performance, high-fidelity, mobile "
+                        "apps for iOS and Android from a single codebase. This "
+                        "gallery is a preview of Flutter's many widgets, behaviors, "
+                        "animations, layouts, and more. Learn more about Flutter at "
+                      ),
+                      new LinkTextSpan(
+                        style: aboutLinkStyle,
+                        url: 'https://flutter.io'
+                      ),
+                      new TextSpan(
+                        style: aboutTextStyle,
+                        text: ".\n\nTo see the source code for this app, please visit "
+                      ),
+                      new LinkTextSpan(
+                        style: aboutLinkStyle,
+                        url: 'https://goo.gl/iv1p4G'
```

Comment:
I'm surprised you're using a shortened link instead of just https://github.com/flutter/flutter/tree/master/examples/flutter_gallery.  Does the shortened link allow you to change it after the fact or some other value-add?